new/PRS12786Yohannes.xml

```diff
@@ -61,7 +61,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                     <floruit/>
                 </person>
                 <listRelation>
-                    <relation name="saws:isRelatedTo" active="PRS12786Yohannes" passive=""/>
+                    <relation name="saws:isRelatedTo" active="PRS12786Yohannes" passive="RNBefns7"/>
```

Comment:
isAttested would maybe be a better relation here?

Reply:
No.